doc/api/http.md

```diff
@@ -300,6 +300,16 @@ added: v0.3.6
 By default set to `Infinity`. Determines how many concurrent sockets the agent
 can have open per origin. Origin is the returned value of [`agent.getName()`][].
 
+### `agent.maxTotalSockets`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+By default set to `Infinity`. Determines how many concurrent sockets the agent
+can have open.
```

Comment:
Maybe add a note along the lines of `across all origins` to clarify the difference with `maxSockets`.

Reply:
Yes, right !